### PR TITLE
add Text Utilities

### PR DESCRIPTION
**Pull request by hsnice16**
added the code for Text Utilities.

Met the following requirements -

image

This is how UI looks,

- Heading
image

- Small Text and Gray Text
image

- Center Text
image

- Display
image

### I have also added some Utilities Class related to fonts

- Font Size
image

- Font Weight
image

- Text Color
image

Please review this PR.

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
Looks good to me.

File: css/root.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 :root {
```
Comment (venkateshwarans) [inline css/root.css:3]: You have only included font-weight 400. So all your font-weight variations look the same. from 100 to 500 and 600 - 900

File: index.html [hunk 2]
```diff
@@ -509,6 +509,111 @@ <h2 class="section-head">Error Form Input</h2>
...
+      </section>
+
+      <!-- Font Weight Ends -->
+
+      <!-- Text Color Starts -->
+
+      <section class="section-component">
+        <h2 class="section-head">Text Color</h2>
+
+        <p class="text-gra-500">color: var(--COLOR-GRAY-500)</p>
```
Comment (venkateshwarans) [inline index.html]: typo

**Review by venkateshwarans — APPROVED** [review]
Overall looks good. Added a couple of comments.

File: css/root.css (hunk 1 above)
Comment (hsnice16) [reply css/root.css:3]: Thank you, @venkateshwarans for letting me know. I have includes all variations that google font has. This is how it looks, 
image

File: index.html (hunk 2 above)
Comment (hsnice16) [reply index.html]: Thank you, for letting me know.

**Pull request merged**